contracts/test/ComputeSlotTests.t.sol

```diff
@@ -0,0 +1,20 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.23;
+
+import {Test} from "forge-std/Test.sol";
+// solhint-disable no-console
+import {console2} from "forge-std/console2.sol";
+
+contract ComputeSlotTest is Test {
+    function testComputeStorageSlot() public {
```

Comment:
Having a test without any assert/require statements seems a little odd. Should we add each of the hard-coded storage slot values here just as a sanity check that they are correct?

Reply:
went with making the storage slots public and checking in unit tests